spec/e2e/pbivizInfoSpec.js

```diff
@@ -44,6 +44,14 @@ describe("E2E - pbiviz info", () => {
         process.chdir(tempPath);
         FileSystem.runPbiviz('new', visualName);
         process.chdir(visualPath);
+
```

Comment:
Do we need to duplicate that code across many files?

Reply:
Will update Ilfat's PR but we have to close this one